ui/src/app/dashboard/total-summary/total-summary.component.css

```diff
@@ -1,18 +1,31 @@
-.mat-card {
+.total-summary{
   margin: 1rem;
 }
 
-.count a {
+.card {
+  display: inline-block;
+  margin: 1rem;
+  width: 18%;
+}
+
+.title {
+  font-size: large;
+}
+
+a {
   text-decoration: none;
   color: #000000;
+  font-size: xx-large;
 }
 
-.mat-card-title {
-  font-size: 1.2rem;
-  color: gray
+clr-tooltip {
+  height: 2.5rem;
+  width: 2.5rem;
+  margin-right: 20%;
+  float: right;
 }
 
-/*TODO(zachx): change this to adapt more complex layout*/
-.card-container {
-  display: inline-flex;
+.icon {
```

Comment:
These icons feel a bit too large based on the mocks which show closer to 15x15px, perhaps 50% instead?

Reply:
I do not think it is too big. We can discuss it later.